unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/ChatDescription.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ChatDescription : MonoBehaviour
+{
+
+    [SerializeField] private ChatHUDView chatView;
+    [SerializeField] private bool analyzeResetPositionOnViewRefresh;
```

Comment:
It can be `false ` for cases in which we know that the content is not being resized (for example, `nearby` channel). In that case, once it has been filled, there is no particular reason to recheck if the position should be reset.

Maybe the reset analysis can be done in every case, just in case in the future this changes and less messages are loaded.

Reply:
yeah i would do it always to simplify the configuration and the code